privacy.md

```diff
@@ -0,0 +1,165 @@
+[privacy policy][BotBot].
+
+The Foreman community uses a forum using the [Discourse](https://discourse.org)
+software. When signing up for a Discourse account, you provide a name and email
+address - and in addition we store the IP address of your most recent
+connection. The email address is only used to send you updates to your forum
+topics, personal messages from other users, and digest summaries. You have full
+control over the emails sent from your Discourse account preferences. The forum
+also uses cookies to store your session.
+
+We also use the IP addresses and *public* post data to provide aggregated
+statistics about the forum community, such as posts-per-month, user engagement,
+etc. We also use the IP addresses to help identify [sock
+puppeting](https://en.wikipedia.org/wiki/Sockpuppet_(Internet)) and spam
+accounts.
+
+* Legal basis for this data usage: Legitimate interest, consent
+
+## Surveys
+
+The Foreman community runs a survey each year. Completing the survey is op-in,
+and providing a contact email address in the survey is *optional*. Such
+addresses are stored until the survey analysis is complete, and then deleted
+from the raw survey data before publication. The survey data is used to gain
+understanding of the community, and aggregated statistics based on the survey
+results are published each year.
+
+* Legal basis for this data usage: Legitimate interest, consent
+
+## Foreman debug archives
+
+The Foreman project provides a tool called [foreman-debug][foreman debug] which
+collects logs from your Foreman server and uploads it to a secure location in
+our infrastructure. Use of this tool is entirely voluntary (it is never run
+automatically), and the data is used by our developers to assist users in
+debugging complex problems. The data is not used for any analysis or
+statistics, is only accessible to a small number of the Foreman developers, and
+is deleted regularly.
+
+* Legal basis for this data usage: Consent
+
+# Your rights regarding your data
+
+## Your choices
+
+### You can choose not to provide us with personal data
+
+Using a VPN, Tor, or other IP / UserAgent masking service will not affect your
+use of the project website, as it is entirely static.
+
+### Your accounts on our services
+
+Your Redmine and Discourse accounts contain options to control what emails are
+sent to you. 3rd party services such as [GitHub](https://github.com) retain
+their own policies and privacy options.
+
+## Your rights
+
+Where the EU General Data Protection Regulation 2016/679 (“GDPR”) applies to
+the processing of your personal data, especially when you access the website
+from a country in the European Economic Area (“EEA”), you have the following
+rights, subject to some limitations, against the Foreman Project:
+
+* The right to access your personal data
+* The right to rectify the personal data we hold about you
+* The right to erase your personal data
```

Comment:
How does the Foreman project plan to handle "Right to erasure" requests that relate to Git commit history?

Will the project rewrite commit history to remove every instance of a user's email and force all other users to force pull/rebase or will the project take a stance similar to [Gitlab](https://about.gitlab.com) and require that a user forfeits their right to erasure on commit history data as it could be seen as detrimental to the project, code base, and overall product to rewrite the commit history?

Reply:
I don't think we should rewrite. There are too many issues with it.

Reply:
I think you can argue both "consent" and "legitimate interest" grounds for not rewriting it.

The user has full control over what is submitted to the Git repo, including the option to use psudeonymous contact information, so providing your real name / email is opt-in, and you're not being denied the service if you don't want to.

We also would like contact information, particularly in the case (as other projects have had) where you need to relicense, and thus contact all the contributors. Also if we need to show provenance of intellectual property for any reason. So there's legitimate interest of the project too.

We do need to be clear that this is the case though, so I'll update the text. Thanks for the heads up

Reply:
This is a nice reference, by the way - https://law.stackexchange.com/a/28858